docs/source/usage/security_recommendations.rst

```diff
@@ -0,0 +1,113 @@
+Security Recommendations
+========================
+
+BIOS Settings Reset
+-------------------
+
+A lessee leasing a node can change the BIOS settings. If these settings are not reset after the node is returned, then this can result in unexpected behavior for future users of the node. Fortunately Ironic allows nodes to `specify BIOS settings`_ that are applied upon cleaning.
+This feature has been tested and the details are listed below.
+
+Prerequisites
+~~~~~~~~~~~~~
+Enable a specific hardware type that supports BIOS configuration and then enable bios interface. `iDrac`_ hardware type is used here.
+
+* Install python-dracclient:
+
+.. prompt:: bash $
+
+  sudo pip install 'python-dracclient>=3.1.0'
+
+* Enable the idrac hardware type with the required interfaces in ironic.conf:
+
+.. prompt::
+
+  [DEFAULT]
+  enabled_hardware_types=idrac
+  enabled_management_interfaces=idrac-wsman
+  enabled_power_interfaces=idrac-wsman
+  enabled_vendor_interface=idrac-wsman
+  enabled_bios_interface=idrac-wsman
+
+* Restart the ironic conductor service for the new configuration to take effect.
+
+Set the Node
+~~~~~~~~~~~~
+The following command sets a baremetal node with the idrac hardware type using WSMAN for all interfaces:
+
+.. prompt:: bash $
+
+  openstack baremetal node set \
+    --driver idrac \
+    --power-interface idrac-wsman \
+    --management-interface idrac-wsman \
+    --vendor-interface idrac-wsman \
+    --console-interface no-console \
+    --bios-interface idrac-wsman \
+    --driver-info drac_username=<drac_user> \
+    --driver-info drac_password=>drac_passwd> \
+    --driver-info drac_address=<drac_ip> \
+    <node>
+
+Cleanup BIOS Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+After the node is returned, owner can retrieve the cached BIOS configuration:
+
+.. prompt:: bash $
+
+  openstack baremetal node bios setting list (-f json) <node>
+
```

Comment:
I think it may make sense to have an example of setting the default BIOS settings here?

Reply:
"openstack baremetal node bios setting list" lists the bios settings here. Should I add an example to show what does default settings look like?

Reply:
I don't think there's a need!